docs: note ES|QL views in Kibana editor autocomplete (#5907) (#6085)
elastic/docs-content: explore-analyze/discover/try-esql.md

- **Data source browser**: lists the data sources of the following types that you can query: **Alias**, **Index**, **Integration**, **Lookup Index**, **Stream**, and **Timeseries**. Selections are inserted into the `FROM` or `TS` command and existing sources stay preserved. When the query starts with `TS`, only time series data sources are listed.
- **Fields browser**: lists fields for the data sources currently in your query and lets you insert one field at a time at the cursor position.

:::{note}
:applies_to: {stack: preview 9.4.0, serverless: unavailable}

> [{{esql}} views](elasticsearch://reference/query-languages/esql/esql-views.md) aren't shown in the data source browser. The autocomplete menu suggests them as `FROM` sources.

Author: ```suggestion
[{{esql}} views](elasticsearch://reference/query-languages/esql/esql-views.md) aren't shown in the data source browser but they're visible through the autocomplete menu suggestions.
```
